[CIVIS-8783] Add Github Actions manual approval feature to docs

rodrigo-roca opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
<!-- A brief description of the change being made with this pull request. What is your motivation for the PR? -->

### Merge instructions
<!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->

- [ ] Please merge after reviewing

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

andrea-mosk commented on content/en/continuous_integration/pipelines/github.md:
```diff
@@ -39,6 +39,8 @@ further_reading:
 
 - **Queue time**: View amount of time workflow jobs sit in the queue before processing
 
+- **Approval wait time**: View amount of time workflow runs and workflow jobs wait for a manual approval
```
```suggestion
- **Approval wait time**: View amount of time workflow runs and workflow jobs wait for manual approvals
```

andrea-mosk commented on content/en/continuous_integration/pipelines/_index.md:
```diff
@@ -155,21 +155,22 @@ If your CI provider is not supported, you can try setting up Pipeline Visibility
+| {{< ci-details title="Custom tags and metrics at runtime" >}}Support for adding <a href="https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/">user-defined text and numerical tags</a> to pipelines and jobs in CI Visibility.{{< /ci-details >}}                                                                                                               |            ...
+| {{< ci-details title="Parameters" >}}Support for adding custom pipeline parameters that users set (for example, <code>DYNAMICS_IS_CHILD:true</code>). You can then search using these parameters in the <a href="https://docs.datadoghq.com/continuous_integration/explorer/?tab=pipelineexecutions">CI Visibility Explorer</a> to find all events with a specific parameter.{{< /ci-details >}} |            ...
+| {{< ci-details title="Pipeline failure reason" >}}Identification of a specific reason behind a pipeline or job failure.{{< /ci-details >}}                                                                                                                                                                                                                                                       |            ...
 
```
Let's keep the same syntax as before? I would expect just a {{< X >}} change here

rodrigo-roca replied:
oh crap my IDE did some nasty things

alai97 commented on content/en/continuous_integration/pipelines/_index.md:
```diff
@@ -161,8 +161,9 @@ If your CI provider is not supported, you can try setting up Pipeline Visibility
-| {{< ci-details title="Manual steps" >}}Identification of when there is a job with a manual approval phase in the overall pipeline.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} |  |  | {{< X >}} |  |  |  {{< X >}} |
+| {{< ci-details title="Manual steps" >}}Identification of when there is a job with a manual approval phase in the overall pipeline.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  | {{< X >}} |  |  |  {{< X >}} |
 | {{< ci-details title="Queue time" >}}Identification of the amount of time for which a pipeline or job was in the queue before execution.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  |  {{< X >}} |
+| {{< ci-details title="Approval wait time">}}Identification of the amount of time for which a pipeline or job have been waiting for a manual approval.{{< /ci-details >}} |  |  |  |  |  {{< X >}}  |   |   |  |  |  |
```
```suggestion
| {{< ci-details title="Approval wait time">}}Identification of the amount of time for which a pipeline or job has been waiting for a manual approval.{{< /ci-details >}} |  |  |  |  |  {{< X >}}  |   |   |  |  |  |
```

alai97 reviewed (approved):
LGTM

alai97 commented:
Feel free to ping me on Slack or leave a comment here when you want this merged @rodrigo-roca

rodrigo-roca commented:
> Feel free to ping me on Slack or leave a comment here when you want this merged @rodrigo-roca

@alai97 Now this is ready to merge, thanks for the review! I just moved up the row I added one position up.
